### ISSUE TITLE
desaturation factor

### ISSUE THREAD

**Issue by WyattLiu** [alecjacobson/computer-graphics-raster-images#32]
The saturation of two example files does not agree (desaturated.png desaturated.ppm.png), which one should we mimic? It is fairly trivial to get close to desaturated.png but when I try to get desaturated.ppm.png with a change in math, I ended up with very desaturated pictures.

**Comment by bod11** [comment]
I am also confused about how to apply the saturation factor. I would assume, with it being a factor, that you are simply supposed to multiply the saturation by it. However, when I do this, my image becomes way more de-saturated than the image online, and slightly more de-saturated than the ppm.png file.

**Comment by bod11** [comment]
I think I figured out what it is supposed to be, there was a closed issue that talked about it. I think we are supposed to be reducing the saturation by the factor, not reducing the saturation to the factor. So reducing the saturation by 25% and not reducing the saturation to 25%. 

It seems the initial image on the website, and the one in the folder labeled ppm.png, were reduced to 25%. This was actually changed and the image on the site was updated. I think the old image in the folder isn't supposed to be our goal, and instead the updated image on the website that is the more saturated of the two.
